pkg/admission/webhook/generic_webhook_test.go

```diff
@@ -115,9 +115,6 @@ func TestDispatch(t *testing.T) {
 				admission.Create,
 			),
 			cluster: "root:org:dest-cluster",
-			expectedHooks: []webhook.WebhookAccessor{
```

Comment:
Need to keep this. The proper "fix" is to correct the incorrect spec.reference.workspace.path value "foo" - it needs to be "root:org:source-cluster"

Reply:
fixed